packages/backend/src/core/transaction-count/MaterializedViewRefresher.ts

```diff
@@ -33,9 +36,69 @@ export class MaterializedViewRefresher {
   async update() {
     this.logger.info('Refresh started')
     await this.blockTransactionCountRepository.refreshDailyCounts(
-      this.blockProjectIds,
+      this.blockTransactionCounters.map((c) => c.projectId),
     )
     await this.zksyncTransactionRepository.refreshDailyCounts()
     this.logger.info('Refresh finished')
+    await this.checkIfFullySynced()
+  }
+
+  private async checkIfFullySynced() {
+    const lastHour = this.clock.getLastHour()
+    const startOfDay = lastHour.toStartOf('day')
+    const timeToCheck = lastHour.equals(
```

Comment:
Shouldn't it be gte? 
```suggestion
    const timeToCheck = lastHour.gte(
```